x-pack/plugins/ml/server/routes/schemas/data_analytics_schema.ts

```diff
@@ -60,6 +60,14 @@ export const analyticsIdSchema = schema.object({
   analyticsId: schema.string(),
 });
 
+export const deleteDataFrameAnalyticsJobSchema = schema.object({
+  /**
+   * Analytics Destination Index
+   */
+  deleteTargetIndex: schema.maybe(schema.boolean()),
```

Comment:
For consistency with the other schemas for data frame analytics, I think these should be `deleteDestIndex` and `deleteDestIndexPattern`

Reply:
Renamed in https://github.com/elastic/kibana/pull/66934/commits/5bd97e9c466ad1a88d32da4f432583d5ddc3aed0